### DISCUSSION TITLE
xrepo安装完依赖后一个小疑问

### DISCUSSION THREAD

**Discussion by Keeyuu** [xmake-io/xmake#2460] [General]
在项目里面引入依赖后，vscode怎么获得头文件的代码提示，如果直接把安装路径加到头文件搜索路径里面感觉太长了，还是我的打开方式不对，漏了

**Comment by waruqi** [comment]
看下，https://github.com/xmake-io/xmake-vscode/issues/40

**Reply by Keeyuu** [reply]
解决了
